src/ghdcbot/engine/identity_linking.py

```
@@ -0,0 +1,178 @@
+        self._github = github_identity
+        self._ttl = timedelta(minutes=ttl_minutes)
+
+    def create_claim(self, discord_user_id: str, github_user: str, *, max_age_days: int | None = None) -> LinkClaim:
+        code = _generate_verification_code()
+        expires_at = datetime.now(timezone.utc) + self._ttl
+        # Ensure schema exists for identity_links before writing.
+        try:
+            self._storage.init_schema()
+        except Exception:  # noqa: BLE001
+            # init_schema is idempotent; ignore failures here as they'll surface on insert.
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)